packages/babel-preset/src/utils/dedupeCSSRules.test.ts

```diff
@@ -0,0 +1,31 @@
+import type { CSSRulesByBucket } from '@griffel/core';
+import { dedupeCSSRules } from './dedupeCSSRules';
+
+describe('dedupeCSSRules', () => {
+  it('dedupes simple rules', () => {
+    const rules: CSSRulesByBucket = {
+      d: ['.foo { color: red; }', '.foo { color: red; }', '.baz { color: pink }'],
+    };
+
+    expect(dedupeCSSRules(rules)).toMatchObject({
+      d: ['.foo { color: red; }', '.baz { color: pink }'],
+    });
+  });
+
+  it('dedupes rules with metadata', () => {
```

Comment:
nit: would be worth having a test case with same rule but different media query+metadata